tools/setup/emoji/emoji_names.py

```diff
@@ -1950,7 +1964,7 @@
     "269c": {"canonical_name": "fleur_de_lis", "aliases": []},
     "26a0": {"canonical_name": "warning", "aliases": ["caution", "danger"]},
     "26a1": {"canonical_name": "high_voltage", "aliases": ["zap"]},
-    "26a7": {"canonical_name": "transgender_symbol", "aliases": ["transgender"]},
+    "26a7": {"canonical_name": "transgender_symbol", "aliases": []},
```

Comment:
Why do these aliases changes happen?

Reply:
I'm pretty sure it's because "transgender" is now an alias for multiple emoji (and therefore no emoji) with the addition of the flag